spring-session/src/main/java/org/springframework/session/MapSession.java

```diff
@@ -40,7 +40,7 @@
  * @since 1.0
  * @author Rob Winch
  */
-public final class MapSession implements ExpiringSession, Serializable {
+public class MapSession implements ExpiringSession, Serializable {
```

Comment:
Thanks @wilkinsona For some reason I thought the Gradle plugin only checked runtime and not optional scopes.

What are your thoughts on adding FST as a dependency to Spring IO Platform?